TextFieldCounter/TextFieldCounter.xcodeproj/project.pbxproj

```diff
@@ -270,7 +273,8 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = serralvo.co.TextFieldCounterTests;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```

Comment:
I think we should turn it off. 
https://stackoverflow.com/questions/44379348/the-use-of-swift-3-objc-inference-in-swift-4-mode-is-deprecated

Reply:
@filletofish yes, agree.

Reply:
@filletofish fix pushed